config/locales/zh-TW.yml

```diff
@@ -0,0 +1,298 @@
+---
```

Comment:
Let's set [the fallback langauge](https://stackoverflow.com/a/7886246/384110) and only include the translated lines in this file.

Reply:
Ok, we have `config.i18n.fallbacks = true` in `application.rb`. I have removed duplications in the zh-TW file.